components/TextFields/src/ContainedInputView/private/MDCBaseTextFieldLayout.h

```diff
@@ -31,6 +31,9 @@
 @property(nonatomic, assign) CGRect leftViewFrame;
 @property(nonatomic, assign) CGRect rightViewFrame;
 
+@property(nonatomic, readonly) CGFloat calculatedHeight;
```

Comment:
if calculatedHeight getter is containerHeight getter, do we need this additional property?

Reply:
After I add assistive labels it will be possible for calculatedHeight to be larger than containerHeight. The implementation here is only temporary.